meta/src/ast.rs

```diff
@@ -254,6 +254,79 @@ impl Expr {
     }
 }
 
+impl core::fmt::Display for Expr {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        match self {
+            Expr::Str(s) => write!(f, "{:?}", s),
+            Expr::Insens(s) => write!(f, "^{:?}", s),
+            Expr::Range(start, end) => {
+                let start = start.chars().next().expect("Empty range start.");
```

Comment:
I think it's a range of character, so it's expected to be a string with exact one character. For example, in [pest_generator::generator::generate_expr()](https://vscode.dev/github/pest-parser/pest/blob/master/generator/src/generator.rs#L389), `next().unwrap()` is used.
So maybe the tuple variant should be `(char, char)`. May I change it? It may be breaking.

Reply:
It can be left, just something I noticed